postgres/datadog_checks/postgres/config.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, instance):
 
         # Deep Database monitoring adds additional telemetry for statement metrics
         self.deep_database_monitoring = is_affirmative(instance.get('deep_database_monitoring', False))
+        self.statement_metric_limits = instance.get('statement_metric_limits', None)
```

Comment:
(Nit) We seem to use the wording "statement metrics" (plural) elsewhere, eg `class PostgresStatementMetrics: ...`. Should we use plural here as well for consistency?

```suggestion
        self.statement_metrics_limits = instance.get('statement_metrics_limits', None)
```